core/livepeernode.go

```diff
@@ -37,6 +37,7 @@ var EthRpcTimeout = 5 * time.Second
 var EthEventTimeout = 5 * time.Second
 var EthMinedTxTimeout = 60 * time.Second
 var DefaultMasterPlaylistWaitTime = 60 * time.Second
+var DefaultJobLength = int64(96)
```

Comment:
I know this is easily changed, but does this mean that the default max job length is 96 blocks (24 minutes)?

Reply:
Oh yeah... Should be `86400/15=5760`.  Good catch.